Makefile

```diff
@@ -22,7 +22,7 @@ generate: install-generator  ## Generate the SDK from our public openapi spec
 		-o ./generated \
 		--additional-properties=packageName=groundlight_openapi_client
 # strict-nullable makes nullable fields Optional in the generated Pydantic classes: https://github.com/koxudaxi/datamodel-code-generator/issues/327
-	poetry run datamodel-codegen  --input spec/public-api.yaml --output generated/model.py --strict-nullable
+	poetry run datamodel-codegen  --input spec/public-api.yaml --output generated/model.py --strict-nullable --use-schema-description
```

Comment:
what does this new flag do?

Reply:
It uses the schema description to generate docstrings on the generated classes